File: _0001.java
public class _0001 {

    public Object fn0() {
        int sum = 0;
        for (int i = 0; i < 1000; i++)
            if (i % 3 == 0 || i % 5 == 0)
                sum += i;
        return sum;
    }

    public String[] solutions = new String[]{"fn0"};
}

File: 0001.cpp
#include <iostream>
#include <vector>
#include <functional>

void fn0() {
  int sum = 0;
  for (int i = 0; i < 1000; i++) {
    if (i % 5 == 0 || i % 3 == 0) {
      sum += i;
    }
  }
  std::cout << sum << std::endl;
}

std::vector<std::function<void()>> progs = { fn0 };
